server/src/handlers/posts/update-post.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express'
+import { Db } from 'mongodb'
+import { ObjectId } from 'mongodb'
+
+/** finds and returns a post by id */
+export const updatePostHandler = (database: Db) => async (
+  req: Request,
+  res: Response,
+) => {
+  const { postId } = req.params
+
+  try {
+    const getPostById = await database
+      .collection('posts')
+      .findOneAndReplace(
```

Comment:
```suggestion
      .findOneAndUpdate(
```

https://stackoverflow.com/a/39155411/7607265